frontend/src/components/common/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import styles from './styles.module.scss';
+
+interface CheckboxType {
+  options: Array<string>;
+  onChange(value: string): void;
+}
+
+export const Checkbox: React.FC<CheckboxType> = ({ onChange, options }) => {
+  return (
+    <div>
+      {
+        options.map(typeOfDoctor => {
```

Comment:
Why type doctor? Common components shouldn't know anything about specific names. They work only with interfaces(in our case, with `IOption`)